config/cruisecontrol.properties

```diff
@@ -165,7 +165,13 @@ num.proposal.precompute.threads=1
 zookeeper.connect=localhost:2181/
 
 # The max number of partitions to move in/out on a given broker at a given time.
-num.concurrent.partition.movements.per.broker=10
+num.concurrent.inter.broker.partition.movements.per.broker=2
+
+# The max number of partitions to move between disks within a given broker at a given time.
+num.concurrent.intra.broker.partition.movements.per.broker=2
+
+# The max number of leadership movement within the whole cluster at a given time.
+num.concurrent.leader.movements=100
```

Comment:
Let's not silently change the value of this config from `1000` to `100` in this patch. In general, in order to improve visibility of such changes, they should not be a part of gigantic monolithic patch -- such behavior changes should have their own PR and justify the reason for change.